.github/workflows/e2e.yml

```
@@ -0,0 +1,156 @@
+        working-directory: backend
+        run: |
+          ./target/release/tsv-tennis-backend &
+          for i in $(seq 1 30); do
+            curl -sf http://localhost:5000/api/health && break
+            sleep 1
+          done
+
+      - name: Start frontend dev server
+        working-directory: tsv-tennis-app
+        run: npm run dev -- --port 5173 &
+        sleep 5
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)